# Request
int8    count
---
# Response
bool    success
string  message